verticapy/machine_learning/feature_extraction/text.py

```diff
@@ -0,0 +1,197 @@
+from verticapy._typing import SQLRelation
+from verticapy._utils._sql._sys import _executeSQL
+from verticapy.core.vdataframe.base import vDataFrame
+
+class Tfidf:
+    """
+    Create tfidf representation of documents.
+
+    Parameters
+    ----------
+    name: str
+        Name of the model.
+    schema: str
+        Name of the model.
+    overwrite_model: bool, default=False
+        If set to True, training a model with the same
+        name as an existing model overwrites the
+        existing model.
+    lowercase: bool, default=True
+        Converts  all  the elements to lowercase  before
+        processing.
+    """
+    def __init__(
+        self,
+        name: str = None,
+        schema: str = None,
+        overwrite_model: bool = False,
+        lowercase: bool = True
+    ) -> None:
+         self.parameters = {
+             "name": name,
+             "schema": schema,
+             "overwrite_model": overwrite_model,
+             "lowercase": lowercase
+         }
+
+    def fit(
+        self,
+        input_relation: SQLRelation,
+        index: str, 
```

Comment:
why not renaming column to x so we unify the fit functions (or X if you consider the possibility to merge multiple columns together in your function).